internal/topology/variables/cluster_variable_validation_test.go

```diff
@@ -27,215 +27,6 @@ import (
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-func Test_ValidateClusterVariables(t *testing.T) {
```

Comment:
Let's please keep the test order the same for this PR. It just makes the diff unnecessary hard 
(also not sure why the order was changed)